rules-tests/Transform/Rector/Class_/AddAllowDynamicPropertiesAttributeRector/config/configured_rule.php

```diff
@@ -7,5 +7,12 @@
 
 return static function (ContainerConfigurator $containerConfigurator): void {
     $services = $containerConfigurator->services();
-    $services->set(AddAllowDynamicPropertiesAttributeRector::class);
+    $services->set(AddAllowDynamicPropertiesAttributeRector::class)
+        ->call('configure', [[
+            [
```

Comment:
@samsonasik - I know the docs say it should start with `[[` however for some reason when I do that the `configure` method gets the wrong value. By adding a third level of array it's working as expected.
Any thoughts on that?

---

Without the "extra" array I get:
```
array(1) {
  [0] =>
  string(19) "*\Fixture\Process\*"
}
```

With the extra array:
```
array(1) {
  'transform_on_namespaces' =>
  array(1) {
    [0] =>
    string(19) "*\Fixture\Process\*"
  }
}
```

Reply:
Actually, never mind - I see now that there's some sort of magic going on there. Where when the array contains a single member whose value is an array it unwraps to that member.

Reply:
NVM. Got that sorted out by looking at existing rules.